### PR TITLE
Hyperlink to conduct on Community page

### LINKED ISSUE #730: [Community] Hyperlink to full-length code of conduct on Community page
**Current Behavior**
The https://layer5.io/community page has a section describing our code of conduct:

```

Code of Conduct
The comfort and safety of Layer5 community members is our priority. 
You must agree to the Code of Conduct to participate in the Layer5 community, 
and any violations of the Code of Conduct will be taken seriously. To report any 
violations please fill out this incident form.

```

**Desired Situation**
In the second sentence of the paragraph above, the text "Code of Conduct" should have the Layer5 Code of Conduct hyperlinked and available to view. Hyperlink: `/terms/code-of-conduct`

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### LINKED ISSUE #751: [Community] Style MeshMate cards similarly to member cards
**Description**
The cards on this page - https://layer5.io/community/meshmates - need to change color on hover like the member cards on https://layer5.io/community/members.

**Resources**
- [Collection of members](https://github.com/layer5io/layer5/tree/master/collections/_members)
- [Relevant style](https://github.com/layer5io/layer5/blob/master/assets/css/materialize.css#L2868-L2892) (some but maybe not all of it) 

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by NorbertLuszkiewicz**
Signed-off-by: NorbertLuszkiewicz <norbertluszkiewicz00@gmail.com>

**Description**

  I added hyperlink to full-length code of conduct on Community page

This PR fixes #730 
Fixes #751 

 

**Notes for Reviewers**

  https://github.com/layer5io/layer5/issues/730

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Tanuj22** [comment]
@NorbertLuszkiewicz thanks for jumping in. There is a merge conflict can you look into it?

**Comment by leecalcote** [comment]
Thank you, @Jashpatel1. I went ahead and resolved the conflict. 👍

File: _includes/community/meshmates-program.html [hunk 1]
```diff
@@ -1,63 +1,115 @@
...
   <p class="indent">
-    The comfort and safety of Layer5 community members is our priority. You must agree to the Code of Conduct to participate in the Layer5 community, and any 
-    violations of the Code of Conduct will be taken seriously. To report any violations please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSeWzC5HjlHugFjB0TtaAVnSkPPqsRQ3JRYjdwyDXf0oyRxcdQ/viewform">this incident form</a>.
+    The comfort and safety of Layer5 community members is our priority. You must
+    agree to the Code of Conduct to participate in the Layer5 community, and any
+    violations of the <a href="/terms/code-of-conduct">Code of Conduct</a> will
+    be taken seriously. To report any violations please fill out this 
+    <a href="https://docs.google.com/forms/d/e/1FAIpQLSeWzC5HjlHugFjB0TtaAVnSkPPqsRQ3JRYjdwyDXf0oyRxcdQ/viewform">
+    this incident form</a>.
+    form.
```
Comment (Tanuj22) [inline _includes/community/meshmates-program.html:30]: the text is "this incident form". The "form" part is repeating twice

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Just some minor changes if you would look into them @NorbertLuszkiewicz 

Screenshot from 2020-09-10 01-49-15
The color should be gray only for hover.

**Comment by Nikhil-Ladha** [comment]
Also, the MeshMate page is not responsive. Can you pleasese fix that too?

**Comment by Tanuj22** [comment]
> Also, the MeshMate page is not responsive. Can you pleasese fix that too?

Nice work here @NorbertLuszkiewicz :100: .Can you also have a look at this^^

**Comment by NorbertLuszkiewicz** [comment]
Looking responsive
 img
 img

**Comment by Nikhil-Ladha** [comment]
Screenshot from 2020-09-11 12-14-39
This is for the tablet view.

**Comment by NorbertLuszkiewicz** [comment]
ok I'll take care of it

**Review by leecalcote — APPROVED** [review]
Thank you, @NorbertLuszkiewicz. 👍

**Pull request merged**